readme.md

```diff
@@ -14,6 +14,17 @@ To install the Jetpack plugin on your site, [follow the instructions on this pag
 
 Do you need help installing Jetpack, or do you have questions about one of the Jetpack modules? You can [search through our documentation here](http://jetpack.com/support/). If you don't find the answers you're looking for, you can [send us an email](http://jetpack.com/contact-support/) or [start a new thread in the WordPress.org support forums](https://wordpress.org/support/plugin/jetpack#postform).
 
+## Installation from  git repo
+
+The `master-stable` branch of this repo contains an stable version with every JS file pre-built.
+
+```
+$ cd wp-content/plugins
+$ git clone https://github.com/Automattic/jetpack
```

Comment:
I'd recommend using `git@github.com:Automattic/jetpack.git` instead, so you don't need a password every time you want to push.